libs/hdf-converters/src/mappings/NiktoNistMapping.ts

```diff
@@ -1,34 +1,20 @@
-import {default as data} from '../../data/nikto-nist-mapping.json';
-import {NiktoNistMappingItem} from './NiktoNistMappingItem';
+import {data} from './NiktoNistMappingData';
 
 export interface INIKJSONID {
   'NIKTO-ID': number;
   'PLUGIN-CATEGORY': string;
   'NIST-ID': string;
   OSVDB: number;
 }
-const DEFAULT_NIST_TAG = ['SA-11', 'RA-5'];
+const DEFAULT_NIST_TAG = ['AC-3', 'SA-11', 'RA-5'];
```

Comment:
It is unclear why this DEFAULT was changed to add AC-3. I can't find an issue on this.